hs_discover/src/components/Search.vue

```diff
@@ -210,19 +222,28 @@
                             <img v-if="entry.geo" src="/static/img/Globe-Green.png" height="25" width="25" v-b-tooltip.hover title="Contains Spatial Coverage">
                             </span>
                         </td>
```

Comment:
can move styling in this case to classes (in many other parts of HS I am keeping inline as a part of knowing where/what to consolidate)